trestle/core/resolver/_import.py

```diff
@@ -81,6 +88,19 @@ def __init__(
                     f'Back matter resource resolution needed for profile import failed with error: {str(e)}'
                 )
 
+        uri_type = cache.FetcherFactory.get_uri_type(self._import.href)
+        # if this looks like a relative path to remote source, append parent path
+        if uri_type == cache.FetcherFactory.UriType.LOCAL_FILE and self._parent_url_root:
+            self._import.href = self._parent_url_root + self._import.href
+        # if href is now a remote path, capture its parent path for possible use with child imports that are relative
+        if uri_type not in [cache.FetcherFactory.UriType.LOCAL_FILE, cache.FetcherFactory.UriType.TRESTLE]:
```

Comment:
Just in case we put other Uri types in the future, it is better to say for which cases it does apply rather than the opposite as it will be incorrect

Reply:
That would of course also fail if the new types were then missed.  I have made an explicit function defined right next to the enums so it should be more manageable and abstracted.